pages/profile.js

```diff
@@ -2,18 +2,16 @@ import Layout from "../components/layout";
 import { useAuth0 } from "@auth0/auth0-react";
 
 
-function classNames(...classes) {
-  return classes.filter(Boolean).join(" ");
-}
-
 export default function Profile() {
   const { user, isAuthenticated, isLoading } = useAuth0();
 
   if (!user) {
+    // TODO TD082321: redirect to login
```

Comment:
There should be a redirect around the "main" application that does not include /legal, /about, and / which redirects if not logged in.

If logged in / should redirect to feed

/ should be a login/sign up page, obviously, with links to about and legal.